packages/manager/modules/pci/src/projects/projects.routing.js

```diff
@@ -88,6 +86,48 @@ export default /* @ngInject */ ($stateProvider) => {
         OvhApiCloudProject.v6().delete({ serviceName: project.serviceName })
           .$promise,
 
+      user: /* @ngInject */ (SessionService) => SessionService.getUser(),
+
+      isTrustedZone: /* @ngInject */ (pciFeatures) => {
+        return pciFeatures.isFeatureAvailable(PCI_FEATURES.OTHERS.TRUSTED_ZONE);
+      },
+
+      /**
+       * This function can be used to know if feature is accessible or not
+       * @param pciFeatures { ovhFeatureFlipping }: feature flipping instance
+       * @param projects { Array }: projects instance list
+       * @returns {(function(*=): (string|boolean))|*}: state where the redirection is going, or false which mean no redirection required
+       */
+      pciFeatureRedirect: /* @ngInject */ (pciFeatures, projects) => (
+        feature,
+      ) => {
+        if (pciFeatures.isFeatureAvailable(feature)) {
+          return projects.length ? 'pci.projects' : 'pci.projects.onboarding';
```

Comment:
looks incomplete, may be you can add TODO comment

Reply:
Idea here is to check for passed feature, the customer is or not able to access to this feature.
Case `customer can't access` -> redirect to projects page or on onboarding page
Case `customer can access` -> no redirection.

Maybe something else ?

Reply:
Ok got it. You are redirecting to projects or onboarding page if customer has access, feature is enabled, is it correct?

Reply:
Customer will be redirect if he don't has feature-flipping access.
If he had a projects, it will redirect to Projects page
if he don't have a Projects, he will be redirect to onboarding page